src/lib/y2storage/filesystems/tmpfs.rb

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/filesystems/base"
+
+module Y2Storage
+  module Filesystems
+    # Class to represent a tmpfs.
+    #
+    # The Tmpfs object should always have MountPoint as child. So the Tmpfs
+    # must be deleted whenever the MountPoint is removed.
+    #
+    # This a wrapper for Storage::Tmpfs
```

Comment:
This *is*.

Reply:
This was copy&pasted from `Nfs`. So you found two typos at once. :wink: Commit with the fix added.